ui/raidboss/data/05-shb/raid/e9n.js

```diff
@@ -229,9 +229,32 @@ export default {
     },
     {
       'locale': 'fr',
-      'missingTranslations': true,
       'replaceSync': {
         'Cloud Of Darkness': 'Nuage de Ténèbres',
+        'Stygian Arbor': 'Arbre ténébreux',
+      },
+      'replaceText': {
+        '(?<!\\w )Particle Beam': 'Rayon explosif',
+        'Anti-Air Particle Beam': 'Faisceau de particules antiaérien',
+        'Bad Vibrations': 'Vibration intense',
+        'Deluge Of Darkness': 'Grand déluge de Ténèbres',
+        'Earth-Shattering Particle Beam': 'Faisceau de particules fracassant',
+        'Empty Plane': 'Cieux obscurs',
+        'Evil Seed': 'Tir semant',
+        'Flood Of Emptiness': 'Déluge de Ténèbres : Cieux obscurs',
+        'Flood Of Obscurity': 'Déluge de Ténèbres : Forêt obscure',
+        'Ground-Razing Particle Beam': 'Faisceau de particules radiant',
+        'Hypercharged Condensation': 'Aspiration particulaire',
+        'Mire Of Despair': 'Marais de Ténèbres',
+        'Obscure Woods': 'Forêt obscure',
+        'Particle Concentration': 'Rayon sphérique',
+        'Rejuvenating Balm': 'Tir vivifiant',
+        'Summon': 'Invocation',
+        'The Art Of Darkness': 'Arts ténébreux',
+        'Waste Away': 'Dépérissement',
+        'Wide-Angle Particle Beam': 'Faisceau de particules élargi',
+        'Wide-Angle Phaser': 'Faisceau de particules bondissant élargi',
+        'Zero-Form Particle Beam': 'Faisceau de particules zéro absolu',
```

Comment:
This conflict in my PR #2306 
I am waiting for Akurosia because there are something might wrong in Deutsch.

Reply:
Oh sorry. I don't saw. I close this request.